### DISCUSSION TITLE
Ability to re-run failed tests and list tests that have failed in editor-friendly form

### DISCUSSION THREAD

**Discussion by jeetsukumaran** [CleanCut/green#6] [Enhancements]
- re-run the last failed tests like nose does.
- list all the failed/error tests in the previous run in way that makes it easy to pass to an editor (perhaps a flag to switch between the dot-as-path form for the test runner and normal slashes + ".py" extension for the editor)

**Comment by Julian** [comment]
I have a similar comment here as in #38, which is that https://pypi.python.org/pypi/testrepository already exists and does this, so if you have #34, you'd be able to just use that, or for me, I'd be able to just keep using that
